pom.xml

```diff
@@ -35,6 +42,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>net.openhft</groupId>
```

Comment:
you may have to update Project.java in the Root pom to ensure that Posix is built after core